src/CloudFoundry.CloudController.Common/Http/SimpleHttpClient.cs

```diff
@@ -120,25 +120,36 @@ public void Dispose()
         /// <inheritdoc/>
         public async Task<SimpleHttpResponse> SendAsync(IEnumerable<HttpMultipartFormData> multipartData)
         {
-            var httpContent = new MultipartFormDataContent();
+            var boundary = Guid.NewGuid().ToString("N") + Guid.NewGuid().ToString("N");
```

Comment:
String format would make it more hard to read. I can't think of a more simple  way for generating non-crypto random string.